CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ initialized=true when a Raft join is in progress. [[GH-10498](https://github.com
 * secrets/database/influxdb: Fix issue where not all errors from InfluxDB were being handled [[GH-10384](https://github.com/hashicorp/vault/pull/10384)]
 * secrets/database/mysql: Fixes issue where the DisplayName within generated usernames was the incorrect length [[GH-10433](https://github.com/hashicorp/vault/pull/10433)]
 * secrets/database: Sanitize `private_key` field when reading database plugin config [[GH-10416](https://github.com/hashicorp/vault/pull/10416)]
+* secrets/gcp: Fix issue with account and iam_policy roleset WALs not being removed after attempts when GCP project no longer exists [[GH-10759](https://github.com/hashicorp/vault/pull/10759)]
```

Comment:
Should this be in https://github.com/hashicorp/vault/tree/master/changelog ?

Reply:
We can remove this entry now that it's in `changelog/10759.txt`.